modules/platforms/dotnet/Apache.Ignite.Core.Tests/TestUtils.cs

```diff
@@ -77,8 +77,8 @@ public static class TestUtils
             ? new List<string>
             {
                 "-XX:+HeapDumpOnOutOfMemoryError",
-                "-Xms2g",
-                "-Xmx4g",
+                "-Xms4g",
+                "-Xmx7g",
```

Comment:
Why do we increase default memory limits in test?

Reply:
I've seen some intermittent failures caused by OOM on Java side while running tests. This is an attempt to improve stability.  If not, we should investigate increased memory usage. I'm keeping an eye on it.